### ISSUE TITLE
cpanm fatpack doesn't run on 5.8.1

### ISSUE THREAD

**Issue by miyagawa** [miyagawa/cpanminus#465]
```
> PLENV_VERSION=5.8.1 cpanm --test-only Carp   
Exporter version 5.57 required--this is only version 5.567 at /loader/0x7fb260c1b920/CPAN/Meta/Check.pm line 1
BEGIN failed--compilation aborted at /loader/0x7fb260c1b920/CPAN/Meta/Check.pm line 1.
Compilation failed in require at /loader/0x7fb260c1b920/App/cpanminus/script.pm line 94.
```

The fatpacked CPAN::Meta::Check uses Exporter 5.57 which is higher than the one shipped with perl < 5.8.3. Need to fatpack Exporter.pm, Exporter/Heavy.pm and newer Carp.pm I think.

**Comment by miyagawa** [comment]
Fixed in CPAN-Meta-Check-0.012 to drop the hard requirement of Expoter 5.57 by @leont

**Comment by miyagawa** [comment]
Just for the record this has been already addressed in 529f810083abbd28476e4978fba9996147b4192f but I was running an outdated copy of cpanm in 5.8.1 :/
